ProjectSidewalk/SidewalkWebpage
# 1454 rename utilitiespanomarker file

Fixes #1454 

We reference `UtilitiesPanomarker.js` in the code, but the file was accidentally named `UtilitiesPanoMarker.js` (the 'M' was capitalized by mistake). This PR just renames that file.

Testing instructions:
1. On the develop branch, load the admin page. You should see some error in the browser console about this file.
1. On the 1454 branch visit the admin page, you should not see that error.
